src/utils/ETools/audio.cpp

```diff
@@ -258,7 +257,7 @@ int aiff_open(FILE* in, oe_enc_opt* opt, unsigned char* buf, int buflen)
     format.samplesize = READ_U16_BE(buffer + 6);
     format.rate = (int)read_IEEE80(buffer + 8);
 
-    aiff->bigendian = 1;
+    int bigendian = 1;
```

Comment:
Was wrong a bit. Let's make it 100% correspond to `aifffile::bigendian` type
```suggestion
    short bigendian = 1;
```